### PR TITLE
Update contact setup in 2.4

### PR DESCRIPTION
**Pull request by mrtuvn**
## Purpose of this pull request

This pull request (PR) update content to avoid confusing for user in 2.4 setup basic contact page

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/stores/contact-us.html

## Links to Magento source code or PRs

- ...
- ...

## Additional information

https://github.com/magento/merchdocs/issues/648

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/stores/contact-us.md [hunk 1]
```diff
@@ -70,51 +70,9 @@ The Luma sample data includes a _Contact Us Info_ block that can be customized f
 
 ### Method 2: Without sample data
 
-1. Create a CMS page for the Contact Us form:
+{:.bs-callout-warning}
+From update in magento 2.4.0 contact form no longer able to call inside cms block or cms page. All custom contact form should be done via layout xml or custom theme templates
 
-    - Follow the standard instructions to create a [new page]({% link cms/page-add.md %}), with the following settings:
+Default contact form can access via contact link in footer webpage. To learn more about customise contact page, see [Devdocs Frontend Guide][theme-guide]{:target="_blank"} in the Magento Devdocs.
```
Comment (leslietilling) [inline src/stores/contact-us.md:76]:
```suggestion
By default, shoppers access the contact form using the Contact link in the footer of the storefront page. For more information about customizing the contact page, refer to our [Frontend Developer Guide][theme-guide]{:target="_blank"}.
```

**Comment by mrtuvn** [comment]
@leslietilling thanks for your suggestion! Updated my PR

**Comment by mrtuvn** [comment]
@bluedarter2 updated docs base on your report

**Comment by leslietilling** [comment]
running tests

**Pull request merged**